### PR TITLE
Package collections landing page stub

### PR DESCRIPTION
**Pull request by finestructure**
Merge after #1148

### CODE REVIEW

**Comment by daveverwer** [comment]
Added a draft of a landing page article. Thoughts welcome!

File: Resources/Markdown/package-collections.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+page-title: Package Collections
+description: The Swift Package Index supports the generation of package collections.
+---
+
+## Package Collections
+
+Package Collections are a new feature in the [Swift 5.5 release](XXX-LINK-TO-SWIFT-55-RELEASE-POST-XXX) of the Swift Package Manager allowing packages to be grouped together, searched, and have their metadata inspected.
```
Comment (daveverwer) [inline Resources/Markdown/package-collections.md:8]: Excuse the passive voice here. Struggling to remove it.

File: Resources/Markdown/package-collections.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+page-title: Package Collections
+description: The Swift Package Index supports the generation of package collections.
+---
+
+## Package Collections
+
+Package Collections are a new feature in the [Swift 5.5 release](XXX-LINK-TO-SWIFT-55-RELEASE-POST-XXX) of the Swift Package Manager allowing packages to be grouped, searched, and have their metadata inspected.
```
Comment (finestructure) [inline Resources/Markdown/package-collections.md:8]: I believe this link (and the one below which I'll mark in a second) are the only pending items. Maybe we should defer these so we can take the url public and update the links once Swift 5.5 and collections are released, avoiding the circular reference.

File: Resources/Markdown/package-collections.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+The default output is plain text, but the command can output JSON if you add the `--json` parameter to the `describe` subcommand.
+
+Finally, if you'd like to remove this package collection, call `swift package-collection remove`:
+
+```
+swift package-collection remove https://swiftpackageindex.com/apple/collection.json
+```
+
+For more information about Swift Package Collections, [see the Apple documentation](XXX-LINK-TO-APPLE-DOCUMENTATION-XXX).
```
Comment (finestructure) [inline Resources/Markdown/package-collections.md:39]: We could skip this paragraph until the official release, or link to https://forums.swift.org/t/se-0291-package-collections/41905 in the meantime.

Comment (daveverwer) [reply Resources/Markdown/package-collections.md]: Yep, I had exactly that thought last night. 

Just removed both links and will queue up a PR to put them back that we can merge whenever we can link to some documentation and the announcement of Swift 5.5.

**Review by daveverwer — APPROVED** [review]

**Comment by daveverwer** [comment]
Tagging as 2.38.2.

**Pull request merged**